skin.json

```diff
@@ -146,6 +146,9 @@
       "height": 81
     }
   },
+  "live": {
+    "dvrEnabled": false
```

Comment:
Why are we deciding DVR / non-DVR on the skin, rather than having the player to tell the skin whether it is DVR or live content?